spring-cloud-kubernetes-leader/src/main/java/org/springframework/cloud/kubernetes/leader/KubernetesHelper.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2018 to the original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.kubernetes.leader;
+
+import java.util.Map;
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.ConfigMapBuilder;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.KubernetesClient;
+
+/**
+ * @author <a href="mailto:gytis@redhat.com">Gytis Trikleris</a>
+ */
+public class KubernetesHelper {
```

Comment:
Since, this is a class method that is specifc to the leader module and not a generic kubernetes helper, I'd prefer a name like 'LeaderHelper', 'LeaderUtils' (or whatever suffix is most commonly used around. :-)

Reply:
`LeaderKubernetesUtils`? `LeaderUtils` doesn't explain what type of the utils are defined in this class. All of them are specific to Kubernetes.

Reply:
+1